src/Javis.jl

```diff
@@ -699,6 +697,7 @@ function set_object_defaults!(object)
     scaleto(desired_scale)
 end
 
+#todo add stroke/fills here
```

Comment:
Also this doesn't seem to be related to this PR.

Reply:
remove